hackclub.com.yaml

```diff
@@ -552,6 +552,13 @@ bank-wrapped:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+bbss:
+  _vercel:
+  ttl: 600
+  type: TXT
+  values:
+    - vc-domain-verify=bbss.hackclub.com,6c79bb5ffee8bbfd78d2
+
```

Comment:
Please add this verification string to the _vercel record at the top of the file.

Reply:
Okie